dashboard/views.py

```diff
@@ -363,10 +373,10 @@ def get_user_assignment_submission(current_user,assignments_in_course_df, course
     return assignment_submissions
 
 
-def hide_score_on_mute(row):
-    if row['grade_muted'] and row['score'] is not None:
-        return None
-    else:return row['score']
```

Comment:
`hide_Score_on_mute` is a redundant probably did not clean up when pulling only graded that are unmuted that is done as part of cron sql query earlier.